system/lvol.py

```diff
@@ -45,11 +45,11 @@
       Float values must begin with a digit.
       Resizing using percentage values was not supported prior to 2.1.
   state:
```

Comment:
Yeah. I'd originally thought overloading state would be following on from how `apt` does it, but now that you mention it another variable is more flexible: you could create a new, disabled, lv with `state=present active=false`. There's no way in my current version you could do that with just one action. Will update the PR.